docs/content/advanced/backend/entities/index.md

```diff
@@ -1,23 +1,6 @@
 # Create an Entity
-
```

Comment:
Don't remove this empty line. There should be one empty line before and after every heading